refresh/templates/services/watsonconversation/importDependency.swift

```diff
@@ -0,0 +1 @@
+        .Package(url: "https://github.com/watson-developer-cloud/swift-sdk",    majorVersion: 0),
```

Comment:
For dependencies with major version at `0`, I like to specify the minor as well. Whilst there are no rules for `0` versions (the are counted as "in development"), often projects will update the minor on breaking changes.

There is also the question of whether anything should really be a `0` release if other projects are depending on it.